addons/website_sale/models/product_template.py

```diff
@@ -468,6 +470,9 @@ def _get_additionnal_combination_info(self, product_or_template, quantity, date,
         )
 
         price_before_discount = pricelist_price
+        pricelist_item = self.env['product.pricelist.item'].browse(pricelist_rule_id)
+        pricelist_compute_price = pricelist_item.compute_price
```

Comment:
should be inside if condition
Also no need for variable `pricelist_compute_price`